FeatureSimulationCommon.pro

```diff
@@ -0,0 +1,57 @@
+TARGET = FeatureSimulationCommon
```

Comment:
The Qt Creator IDE uses them. They can be safely removed if there is no interest in using the IDE